ij-plugin/src/main/resources/org/jetbrains/research/tasktracker/config/task_content_default.yaml

```diff
@@ -1,37 +1,162 @@
 tasks:
-  - name: hello world
-    description: Write a program that will print 'hello world' to the console.
+  - name: Refactoring course
+    description: "Take the refactoring course, and after that, press the next button."
     language: "KOTLIN"
-    focusFileId: "main"
     files:
-      - filename: main
-        sourceSet: "SRC"
-        id: "main"
-    id: "hello"
-  - name: "Example with several files"
-    description: "In this task, we kindly request you to review three provided files and determine which one is the most readable and which one is the least readable. Files: [a.java](a), [b.java](b), [c.java](c)"
-    language: "JAVA"
-    id: "example"
-    files:
-      - filename: a
-        sourceSet: "SRC"
-        templateFile: "a"
-        id: "a"
-      - filename: b
-        sourceSet: "SRC"
-        templateFile: "b"
-        id: "b"
-      - filename: c
-        sourceSet: "SRC"
-        templateFile: "c"
-        id: "c"
-  - name: "python style checker"
-    description: "Modify the file to be more readable and compliant with PEP 8"
-    language: "PYTHON"
-    id: "style"
-    focusFileId: "main"
-    files:
-      - filename: "main"
+      - filename: "Task"
```

Comment:
It would be very helpful to have utils to generate this file by a course automatically

Reply:
Or put the whole folder and add all files from this folder, what do you think?